.changeset/popular-worms-provide.md

```diff
@@ -0,0 +1,5 @@
+---
+'@commercetools-frontend/application-shell': patch
+---
+
+Eliminate CSS modules styles from Navbar styling
```

Comment:
**nit**: just a suggestion. Totally up to you whether you like to use it or not.

```suggestion
Migrated last bits of CSS modules to CSS-in-JS in the Navbar components for better consistency.
```

Reply:
Nit: since this is an internal refactoring and an implementation detail, we can keep it generic

```suggestion
Simplify navbar styles implementation
```